perf/utils/randb.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <random>
+
+struct RandomByteGenerator {
+	RandomByteGenerator(unsigned min = 0, unsigned max = UCHAR_MAX) :
```

Comment:
Why is it `UCHAR_MAX`?

Reply:
Deleted randb.h file as it useless for this benchmark.